components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/pom.xml

```diff
@@ -56,6 +56,16 @@
             <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
             <artifactId>org.wso2.carbon.identity.organization.management.service</artifactId>
         </dependency>
+        <dependency>
```

Comment:
change the year to year range in license header

Reply:
Fixed by - https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/14/commits/e736d14eba35e5af1166de4d6366455f673920c6